src/device_utils.cc

```diff
@@ -10,7 +10,9 @@
 namespace blas {
 
 // -----------------------------------------------------------------------------
-// set device
+/// @deprecated
```

Comment:
For some deprecated, there is a recommended routine to use, but not here.
I just want to make sure it means this routine should not be used at all.

Reply:
Right, there is nothing to replace this. All routines get the device to use from the queue — which is honestly a much better design than having a global "current device" in CUDA and ROCm, even though each thread has its own current device, so it's thread safe.